controller/src/test/java/io/pravega/controller/fault/ControllerClusterListenerTest.java

```diff
@@ -154,7 +154,7 @@ public void clusterListenerTest() throws Exception {
         clusterListener.stopAsync();
 
         clusterListener.awaitTerminated();
-        Assert.assertFalse(clusterListener.isReady());
+        Assert.assertTrue(clusterListener.isReady());
```

Comment:
Why has this been changed to true? Instead of checking `isReady()` we should be checking `isRunning()` here and assertFalse

Reply:
Updated